docs/provider/oauth/facebook.md

```diff
@@ -10,7 +10,7 @@ Select what kind of app you are going to create.
 
 ![Facebook select](/img/providers/OAuth/facebookselect.png)
 
-After fill name and contact email, you can enter facebook developer dashboard.
+After filling in your name and contacting email, you can enter facebook developer dashboard.
```

Comment:
![image](https://user-images.githubusercontent.com/53544726/194458923-5ea6253b-9b1a-4ee1-8145-3f926bd1cdd8.png)
`contact email` does not need to be modified

Reply:
OK